### PR TITLE
Add "Get Ready for the Day" section in public page

### PR DESCRIPTION
**Pull request by thombudel**
screen shot 2017-03-28 at 15 07 54

### CODE REVIEW

**Comment by thombudel** [comment]
- Add test
- Render on page

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,7 +8,10 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
   <body>
+
```
Comment (arnoFleming) [inline app/views/layouts/application.html.erb]: that are a lot of empty lines we dont really need

File: app/views/pages/attend.html.erb [hunk 2]
```diff
@@ -1 +1,3 @@
 <h1>Attend</h1>
+
+<%= render './shared/get_ready_for_day' %>
```
Comment (arnoFleming) [inline app/views/pages/attend.html.erb]: see comments in #54 about `'./shared'` vs `'shared'`

File: app/views/shared/_top_banner_section.html.erb [hunk 3]
```diff
@@ -1,4 +0,0 @@
-<div class="jumbotron">
```
Comment (arnoFleming) [inline app/views/shared/_top_banner_section.html.erb]: why do you remove this file?

File: spec/views/pages/attend.html.erb_spec.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "pages/attend.html.erb",  type: :view do
+
+    it "Should render get ready for day section" do
+      render "shared/get_ready_for_day.html.erb"
```
Comment (arnoFleming) [inline spec/views/pages/attend.html.erb_spec.rb]: nope, see comments in other PRs

File: app/views/shared/_top_banner_section.html.erb (hunk 3 above)
Comment (thombudel) [inline app/views/shared/_top_banner_section.html.erb]: It shouldnt have been in this branch, now it is in master

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -8,7 +8,7 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
   <body>
-    <%= render './shared/navbar' %>
+    <%= render 'shared/navbar' %>
```
Comment (arnoFleming) [inline app/views/layouts/application.html.erb]: TY!

**Pull request merged**